statistics/cmsketch.go

```diff
@@ -547,3 +556,13 @@ func (c *CMSketch) AppendTopN(data []byte, count uint64) {
 func (c *CMSketch) GetWidthAndDepth() (int32, int32) {
 	return c.width, c.depth
 }
+
+// CalcDefaultValForAnalyze calculate the default value for Analyze.
+// The value of it is count / NDV in CMSketch. This means count and NDV are not include topN.
+func (c *CMSketch) CalcDefaultValForAnalyze(NDV uint64) {
+	if NDV < uint64(len(c.topN)) {
+		c.defaultValue = c.count
+	}
```

Comment:
If `NDV < TopN`, we can set `c.defaultValue` to zero and return immediately since all values should be in `TopN`.

Reply:
Agree with that.